src/Game/ReplayServer.java

```diff
@@ -321,8 +346,24 @@ public boolean doTick() {
 
           if (Replay.isSeeking || Settings.FAST_DISCONNECT.get(Settings.currentProfile))
             Replay.timestamp = timestamp_input;
+          
+					disconnected = true;
         }
-      }
+			}
+			
+			// TODO: this only will skip 1 keys, unsure if there is a replay where more than 1 isaac key
+			// should be skipped from OG replays
+			if (nextIncomingPacket != null && nextIncomingPacket.opcode == 10000
```

Comment:
This is kind of weird b/c it ties the opcode logger functionality (which is expected to only be enabled if the Logging level is set high enough) to replay playback functionality.

Reply:
Made into setting now defaulted to true